wave_lang/kernel/wave/water.py

```diff
@@ -33,6 +34,9 @@
     stream_d,
 )
 
+# Water Python bindings are conditionally available
+# We'll try to import them dynamically when needed
```

Comment:
I don't understand what "conditionally available" means and the general purpose of this comment.